frontend/src/tests/components/Nav/AppNavbar.test.js

```diff
@@ -44,6 +44,24 @@ describe("AppNavbar tests", () => {
         expect(adminMenu).toBeInTheDocument();        
     });
 
+    test("renders correctly empty for admin loadSubject", async () => {
```

Comment:
What is this test doing? It doesn't seem to be testing anything from the page you added.

Reply:
I was told by Seth that since I am just creating an empty placeholder page, I could just have a test for an empty page. Should I just delete the test?

Reply:
Testing that a page renders would be slightly different. You are currently rendering a provider and a router, but not a page.